googler

```diff
@@ -339,6 +403,17 @@ class GoogleParser(HTMLParser.HTMLParser):
 
         self.set_handlers_to('main')
 
+    ### Ignore list ###
+    IGNORE_LIST = [
+        # "People also ask"
+        # Sample query: VPN
+        # Screenshot: https://i.imgur.com/yfcsoQz.png
+        {
+            'tag': 'div',
```

Comment:
As I understand from your notes and a quick look at the codes, you are matching each tag against IGNORE_LIST.
For this case, please ignore the `related-question-pair` class while processing inner/outer div. As I see it, we are never going to hit a situation where `related-question-pair` comes outside a div. And it doesn't show up in every result either. So the extra processing in matching against every tag is redundant in most cases.